__tests__/Unit/pages/issues/Issues.test.tsx

```diff
@@ -35,9 +35,7 @@ describe('Issues Page', () => {
         const buttonElement = getByRole('button', buttonAttribute);
         expect(buttonElement).toBeInTheDocument();
 
-        // button should be disabled when the data is loading
-        const loadingElement = getByText(loadingText);
-        expect(loadingElement).toBeInTheDocument();
```

Comment:
why was this removed?

Reply:
Earlier `fetchIssues()` with empty search term was being called on mount whose loading state would render the loadingText.
This is removed now, so currently there is no loadingText on first load.